plugins/catalog/src/components/EntityLayout/EntityLayout.test.tsx

```diff
@@ -36,6 +36,11 @@ import React from 'react';
 import { Route, Routes } from 'react-router';
 import { EntityLayout } from './EntityLayout';
 
+jest.mock('@backstage/plugin-catalog-react', () => ({
```

Comment:
Ditto re. mocking - in this case we even have the api registry broken out for us already :tada:.